ovirtlago/prefix.py

```diff
@@ -49,6 +50,43 @@
 log_task = functools.partial(log_utils.log_task, logger=LOGGER)
 
 
+def _create_junitxml_fullfilename(
+    default_dir, default_filename, junitxml_file=None
+):
+    """
+        Create the file name for the junitxml output
```

Comment:
Please adopt the docstrings to the convention we use in the code.
You can loop for examples in `lago.prefix`.